src/main/java/edu/hawaii/its/api/groupings/MemberAttributeResults.java

```diff
@@ -0,0 +1,58 @@
+package edu.hawaii.its.api.groupings;
+
+import edu.hawaii.its.api.wrapper.Subject;
+import edu.hawaii.its.api.wrapper.SubjectsResults;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MemberAttributeResults implements MemberResults<MemberResult> {
+
+    private String resultCode;
+    private List<String> invalid;
+    private List<MemberResult> results;
+
+    public MemberAttributeResults() {
+        setResultCode("FAILURE");
+        setResults(new ArrayList<>());
+    }
+
+    public MemberAttributeResults(List<String> invalid) {
+        setResultCode("FAILURE");
+        setInvalid(invalid);
+    }
+
+    public MemberAttributeResults(SubjectsResults subjectsResults) {
+        setResultCode(subjectsResults.getResultCode());
+        setResults(subjectsResults.getSubjects());
+    }
+
+    public String getResultCode() {
+        return resultCode;
+    }
+
+    public List<String> getInvalid() {return invalid; }
+
+    @Override
+    public List<MemberResult> getResults() {
+        return results;
+    }
+
+    private void setResultCode(String resultCode) {
+        this.resultCode = resultCode;
+    }
+
+    private  void setInvalid(List <String> invalid) {
+        this.invalid = invalid;
+    }
```

Comment:
For `getInvalid()`, could you fix the formatting, it should look like this:
```
public List<String> getInvalid() {
    return invalid;
}
```

For setInvalid(), remove the space before `void`. Thanks.